packages/vehicle-rental-overlay/src/VehicleRentalOverlay.story.js

```diff
@@ -34,7 +36,7 @@ const MyCircle = ({ fillColor = "gray", pixels, strokeColor }) => {
       {children}
     </CircleMarker>
   );
-  GeneratedCircle.coreUtils.types.propTypes = {
+  GeneratedCircle.propTypes = {
```

Comment:
@landonreed, this is thing that was causing the problems that you were asking about in https://github.com/opentripplanner/otp-ui/pull/268#issuecomment-883335088